Cython/Compiler/ExprNodes.py

```diff
@@ -14401,6 +14405,7 @@ class AnnotationNode(ExprNode):
     # Doesn't handle the pre PEP-563 version where the
     # annotation is evaluated into a Python Object.
 
+    child_attrs = ["expr"]
```

Comment:
This forces `expr` to be processed correctly in Cython, while the point PEP-563 is that it shouldn't have to be. So may cause problems. (We may revise this though)